src/actions/availability.js

```diff
@@ -0,0 +1,78 @@
+import request from "superagent";
+import axios from "axios";
+
+export const AVAILABILITY_ADDED = "AVAILABILITY_ADDED";
+export const AVAILABILITY_FETCHED = "AVAILABILITY_FETCHED";
+
+const baseUrl = "http://localhost:4000";
+
+// Add availability with AUTH
+function availabilityAdded(entity) {
+  return {
+    type: AVAILABILITY_ADDED,
+    payload: {
+      entity: entity
```

Comment:
no need to do `key: value` if you want the key to be the same as the variable name, you can just do
```
payload: { entity }
```
and it's the same